Vizzuality/half-earth-v3
# Feature/refactor-selected-analysis-layer

## Correct source for species count
### Description
### Testing instructions
Go to an precalculated area and check the number of th e species. It should be the same than the tooltip on the global data scene
### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-594?atlOrigin=eyJpIjoiNjNmZTRmZWVhMzBlNGRkZTlhNjcyNTJhZDc0ODE3ZWEiLCJwIjoiaiJ9
